Gordon360/ApiControllers/HousingController.cs

```diff
@@ -8,9 +8,11 @@
 using Gordon360.Exceptions.ExceptionFilters;
 using Gordon360.AuthorizationFilters;
 using Gordon360.Static.Names;
+using Gordon360.Static.Methods;
 using Gordon360.Models.ViewModels;
 using System.Collections.Generic;
-
+using System;
+using System.Diagnostics;
```

Comment:
Visual Studio grays some of these out as if to say they are unused. We should try deleting them and seeing if everything still works

Reply:
This change has been made and tested successfully